examples/deploy_ecs/dagster.yaml

```diff
@@ -12,6 +12,7 @@ run_launcher:
   class: EcsRunLauncher
   config:
     include_sidecars: true
+    secrets_tag: ""
```

Comment:
Maybe it's time to deprecate the ol' magic 😉 

And just make the default behavior do nothing. We could deprecate it with a warning over the next few releases if we want?